src/2981R0_improving_our_safety_with_a_physical_quantities_and_units_library.md

```diff
@@ -331,7 +331,12 @@ quantity<si::kilo<si::metre>, int> q2 = q1;
 ```
 
 We can't preserve the value of a source unit when we convert
-to a unit with a lower resolution while using an integral representation type for a quantity.
+to a unit with a lower resolution while using an integral representation type for a quantity. 
+In the example above converting 5 metre would result in 0 kilometre if internal conversion is 
+done using regular integer handling. While this could be valid behavior the problem arises when the 
+user expect to be able to convert the quantity back to the original unit without loss of information. 
+So the library should prevent such conversions from happening implicitly, wether the library 
+should offer explicitely marked unsafe conversions for cases is yet to be discussed.
```

Comment:
```suggestion
In the example above, converting 5 metres would result in 0 kilometres if internal conversion is 
done using regular integer handling. While this could be valid behavior, the problem arises when the 
user expects to be able to convert the quantity back to the original unit without loss of information. 
So the library should prevent such conversions from happening implicitly; whether the library 
should offer explicitly marked unsafe conversions for these cases is yet to be discussed.
```